src/pipelines/covariates/google_mobility.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any, Dict, List
+from pandas import DataFrame, concat, merge
+from lib.time import datetime_isoformat
+from lib.utils import grouped_diff
+from .pipeline import CovariatesPipeline
+
+
+class GoogleMobilityPipeline(CovariatesPipeline):
+    data_urls: List[str] = [
+        'https://www.gstatic.com/covid19/mobility/Global_Mobility_Report.csv'
+    ]
+
+    def parse_dataframes(self, dataframes: List[DataFrame], **parse_opts):
+        data = dataframes[0].rename(columns = {
+            'country_region_code': 'country_code',
+            'sub_region_2': 'subregion_2_code',
+            'sub_region_1': 'subregion_1_code'
```

Comment:
I think you are not seeing any matches because these appear to be names, not codes. It's unlikely that the name will be an exact match, so I suggest to do the following: if it's subregion1-level data (e.g. state-level) then make `string_match` the value of `sub_region_1` and `subregion_2_code` null, and if it's subregion2-level data (e.g. county-level) then make `string_match` the value of `sub_region_2`.

The merge strategy should try to fuzzy match the value of `string_match` with subregion1_code/name, subregion2_code/name and `string_regex` from auxiliary.csv, in that order. If that fails, it will try to regex match `string_regex` with `string_match`. I suspect that for a number of these we will have to hack it by manually tweaking the `string_regex`.